test/powershell/Modules/Microsoft.PowerShell.Security/FileCatalog.Tests.ps1

```diff
@@ -416,6 +416,62 @@ Describe "Test suite for NewFileCatalogAndTestFileCatalogCmdlets" -Tags "CI" {
             $catalogPath | Should -Not -Exist
         }
     }
+
+    Context "TestCatalog Executing File Validation Tests"{
+
+        AfterEach {
+            Remove-Item "$script:catalogPath" -Force -ErrorAction SilentlyContinue
+        }
+
+        It "Test-FileCatalog should pass when target file is an executing process" {
+            $processFolder =  (Get-Process -Id $pid).Path
+            $script:catalogPath = "$env:TEMP\TestCatalogExecutingFileValidation.cat"
+            $null = New-FileCatalog -Path "$processFolder" -CatalogFilePath $script:catalogPath -CatalogVersion 2
+            $result = Test-FileCatalog -Path "$processFolder" -CatalogFilePath $script:catalogPath
+            $result | Should -Be "Valid"
+        }
+    }
+
+    Context "TestCatalog File Open Validation Tests"{
+
+        BeforeEach {
+            $null = New-Item -ItemType Directory -Path "$env:TEMP\testCatalog" -Force -ErrorAction SilentlyContinue
+            Set-Content -PassThru "$env:TEMP\testCatalog\test.txt" -Value "Test Data" | Out-Null
+        }
+
+        AfterEach {
+            Remove-Item "$script:catalogPath" -Force -ErrorAction SilentlyContinue
+            Remove-Item "$env:TEMP\testCatalog" -Recurse -Force -ErrorAction SilentlyContinue
+        }
+
+        It "Test-FileCatalog should pass when target file has an open reader with FileMode Open FileAccess Read and FileShare Read" {
```

Comment:
The two tests have identical code. Please use `-Case` parameter for FileShare Read/ReadWrite and remove second test.

Reply:
The tests for Context "TestCatalog Skip Validation Tests" are basically having the same problem you're concerned about here as they only change a single parameter.